src/ServiceManagement/RecoveryServices/Commands.RecoveryServices/PSRecoveryServicesClient/PSRecoveryServicesClient.cs

```diff
@@ -259,6 +261,44 @@ private SiteRecoveryManagementClient GetSiteRecoveryClient()
 
             return siteRecoveryClient;
         }
+
+        /// <summary>
+        /// Gets Vault Upgrade client.
+        /// </summary>
+        /// <param name="location">Resource location.</param>
+        /// <param name="resourceName">Resource Name.</param>
+        /// <param name="resourceType">Resource Type.</param>
+        /// <returns>Vault Upgrade Management client.</returns>
+        private RecoveryServicesVaultUpgradeManagementClient GetVaultUpgradeClient(
+            string location,
+            string resourceName,
+            string resourceType)
+        {
+            string resourceNamespace = Environment.GetEnvironmentVariable("RESOURCE_NAMESPACE");
```

Comment:
@neetus15 do you expect users to set this environment variable? If yes, what is the reason for that?
Also it is quite possible, I might have a environment variable "RESOURCE_NAMESPACE" and I have no intention to use it for VaultUpgrade and if I call this cmdlet, this code will end up using it.